examples/proto/examplepb/echo_service.proto

```diff
@@ -9,11 +9,29 @@ package grpc.gateway.examples.examplepb;
 
 import "google/api/annotations.proto";
 
+// Embedded represents a message embedded in SimpleMessage.
+message Embedded {
+	oneof mark {
+		int64 progress = 1;
+		string note = 2;
+	}
+}
+
 // SimpleMessage represents a simple message sent to the Echo service.
 message SimpleMessage {
 	// Id represents the message identifier.
 	string id = 1;
 	int64 num = 2;
+	oneof code {
+		int64 line_num = 3;
+		string lang = 4;
+	}
+	int64 time = 5;
```

Comment:
Can you remove `time` if you are not using it?
I'd like to keep `SimpleMessage` as simple as possible.

Reply:
Done.